build/common.ps1

```diff
@@ -383,7 +383,7 @@ Function Invoke-DotnetPack {
             if($ReleaseLabel -ne 'Release') {
                 $opts += '--version-suffix', "${ReleaseLabel}-${BuildNumber}"
             }
-
+            $opts += '--serviceable'
```

Comment:
This would actually build all the packages with serviceable attribute including test ones, but I think that's fine